### PR TITLE
Special latejoin message for captain

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

title + gender fix for locale messages

## Why / Balance

saw the JoinNotifyCrew in JobPrototype and tried to implement it

## Technical details

no

## Media

afaik it was something like this in ss13, correct me if i'm wrong

image

the gender locale part

Screenshot_20240909_010525

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- tweak: The captain now have special late join message.

### CODE REVIEW

**Comment by muburu** [comment]
should maybe be for other heads of staff too since they are important

File: Resources/Locale/en-US/game-ticking/game-ticker.ftl [hunk 1]
```diff
@@ -33,7 +33,8 @@ player-first-join-message = Player {$name} joined for the first time.
 # Displayed in chat to admins when a player leaves
 player-leave-message = Player {$name} left.
 
-latejoin-arrival-announcement = {$character} ({$job}) has arrived at the station!
+latejoin-arrival-announcement = {$character} ({$job}) { CONJUGATE-HAVE($entity) } arrived at the station!
+latejoin-arrival-announcement-special = {$character}, The {$job}, { CONJUGATE-HAVE($entity) } stepped aboard the station!
```
Comment (Nairodian) [inline Resources/Locale/en-US/game-ticking/game-ticker.ftl:37]: Just a small thought, but I think, "on deck!" fits better, and perhaps you could even rearrange things to be, "Captain [name], on deck!"

Comment (Emisse) [reply Resources/Locale/en-US/game-ticking/game-ticker.ftl:37]: {$job} {$character} { CONJUGATE-HAVE($entity) } on deck!

Comment (dffdff2423) [reply Resources/Locale/en-US/game-ticking/game-ticker.ftl]: To me, "on deck" implies that the captain has steped onto the station which might not be the case (eg. they are on the arrivals terminal).

**Comment by Just-a-Unity-Dev** [comment]
its "Captain (name) on deck!" because of some military origin. saying "captain is aboard the station" sounds really weird imho

**Comment by lzk228** [comment]
> Captain [name] on deck!

idk, this sentence feels odd without a verb
at least `is on deck` would be good, but still odd for me

**Comment by lanse12** [comment]
image
ss13 message

**Comment by Emisse** [comment]
yea just do that and ill merg

**Comment by lzk228** [comment]
yep done

**Pull request merged**